samples/Calling/src/app/views/CallScreen.tsx

```diff
@@ -34,23 +35,15 @@ export interface CallScreenProps {
   roleHint?: Role;
   /* @conditional-compile-remove(call-readiness) */
   callReadinessOptedIn?: boolean;
+  isTeamsIdentityCall?: boolean;
 }
 
 export const CallScreen = (props: CallScreenProps): JSX.Element => {
-  const {
-    token,
-    userId,
-    callLocator,
-    displayName,
-    /* @conditional-compile-remove(PSTN-calls) */ alternateCallerId,
-    /* @conditional-compile-remove(rooms) */ roleHint,
-    /* @conditional-compile-remove(call-readiness) */ callReadinessOptedIn
-  } = props;
+  const { token, userId, /* @conditional-compile-remove(teams-identity-support) */ isTeamsIdentityCall } = props;
   const callIdRef = useRef<string>();
-  const { currentTheme, currentRtl } = useSwitchableFluentTheme();
-  const isMobileSession = useIsMobile();
+
   const afterCreate = useCallback(
-    async (adapter: CallAdapter): Promise<CallAdapter> => {
+    async <CallAdapterType extends CommonCallAdapter>(adapter: CallAdapterType): Promise<CallAdapterType> => {
```

Comment:
hmm this is very confusing for a sample app to read

Reply:
Agree, then I will use another option in my mind - extract the common piece into a single util function